client_test.go

```diff
@@ -225,7 +224,7 @@ func setupDockerLDAP(t *testing.T) string {
 	require.NoError(t, err)
 
 	customLdifFile := path.Join(tempDir, "custom.ldif")
-	err = os.WriteFile(customLdifFile, []byte(customldif), 0644)
+	err = os.WriteFile(customLdifFile, []byte(customldif), 0o644)
```

Comment:
I haven't seen this filemode before and can't seem to find a reference for it. Are there any changes in permissions here?

Reply:
This was done by one of my formatters automatically. This notation is equivalent and seems to have existed [since go 1.13](https://go.dev/doc/go1.13#:~:text=Octal%20integer%20literals%3A%20The%20prefix,by%20octal%20digits%20remains%20valid.)

Reply:
Ah gotcha, thanks for the link!